### ISSUE TITLE
Support for the Hack Club Sprig

### ISSUE THREAD

**Issue by Vishram1123** [fhoedemakers/pico-infonesPlus#2]
I am trying to get this program to work on the Hack Club Sprig [(schematic here)](https://github.com/hackclub/sprig/blob/main/docs/GROWING_A_SPRIG.md), and while I can flash the rom, I cannot get the display or the controls to work. This is expected, as you stated that this works only on the DualShock, DualSense, and the BUFFALO BGC-FC801, and with a specific pinout. However, could you possibly use this schematic to add support for the Sprig?

**Comment by fhoedemakers** [comment]
Unfortunately i don't think i can be of any help in this.

Since the emulator is build for HDMI output (including sound), the display and audio code should be rewritten for use with a TFT breakout board and a separate audio board. 
De Adafruit TFT ST7735 specified in your link has a resolution of  128x160  pixels. This is too low, since a resolution of 256x240 is needed,

**Comment by Vishram1123** [comment]
Alright, thanks anyway

**Comment by Vishram1123** [comment]
Do you think it would be possible to adapt your PicoSystem handheld version to work with this?

**Comment by fhoedemakers** [comment]
Hi, but the resolution is still too low for that device. (128x160 pixels). The NES needs 256x240 pixels. SInce the PicoSystem also has a resolution of 240x240 pixels, i needed to cut-off 8 pixels to the left and 8 pixels to the right. This is somewhat acceptable. But for only 128 pixels screenwidth, this is way too short. Or do i miss something here?

**Comment by Vishram1123** [comment]
You could downscale by getting the average of every other pixel in the x and y directions, so an image like 
![image](https://user-images.githubusercontent.com/64966832/235274771-2962aa61-b2b9-43d1-95fb-d22a03d23714.png)

**Comment by fhoedemakers** [comment]
Sorry for the late response. Was quite busy. 
This could be an option with the PicoSystem handheld as a starting point, however i do not have the device at hand. Can you be of any help youtself in this?

**Comment by Vishram1123** [comment]
Yes, I have the device and I can help (though I am also in the middle of final exams, so it might take a bit to respond); right now, the PicoSystem rom (expectedly) does nothing on my device, but it does show a blue light on the D1 led.

**Comment by HomousMan** [comment]
Did this end up working?

**Comment by fhoedemakers** [comment]
Short answer: no
